minifi-toolkit/minifi-toolkit-configuration/src/test/resources/TemplateWithFunnel.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><template><description></description><name>TemplateWithFunnel</name><snippet><funnels><id>e2c5b2f0-6740-4d57-a40b-20b27da784a3</id><parentGroupId>5e4f1cc2-78e9-4e08-a0c2-11779c9efc17</parentGroupId><position><x>2268.1912159884546</x><y>695.3930796314235</y></position></funnels></snippet><timestamp>09/21/2016 19:08:15 EDT</timestamp></template>
```

Comment:
This and the other three test resources should have the license header.  RAT is currently failing.